<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSudTeilen</class>
 <widget class="QDialog" name="DialogSudTeilen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sud teilen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_Name1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Name2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_Percent1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="horizontalSlider_Percent">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="spinBox_Percent2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Amount1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Amount2">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Liter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Liter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSudTeilen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSudTeilen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
